.github/workflows/elixir.yml

```diff
@@ -4,20 +4,33 @@ on: [push, pull_request]
 
 jobs:
   build:
-    name: Elixir ${{matrix.elixir}} (Erlang/OTP ${{matrix.otp}})
+    name: Test (${{matrix.elixir}}/${{matrix.otp}})
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp: ['22.3']
-        elixir: ['1.11.2', '1.10.4', '1.9.4', '1.8.2', '1.7.4', '1.6.6']
+        otp: [24.x, 25.x, 26.x]
+        elixir: [1.12.x, 1.13.x, 1.14.x, 1.15.x, 1.16.x]
```

Comment:
Any reason to drop support for OTP 22.x, Elixir 1.6 - 1.11?

Reply:
I don't think there is any harm to include older version if it does not impede future supported versions. Elixir 1.11 is not "supported" anymore: https://hexdocs.pm/elixir/1.16.2/compatibility-and-deprecations.html 

Since upgrading Elixir should be very simple, it should be acceptable for new libraries releases to only support from 1.12.x. 

I will add Elixir 1.11 to see if it passes the tests.

Reply:
Adding 1.11 gives the following compilation errors: 
```
== Compilation error in file lib/nimble_parsec/compiler.ex ==
Error: ** (CompileError) lib/nimble_parsec/compiler.ex:931: cannot find or invoke local _/1 inside match. Only macros can be invoked in a match and they must be defined before their invocation. Called as: _(/ / _ = range)
could not compile dependency :nimble_parsec, "mix compile" failed. You can recompile this dependency with "mix deps.compile nimble_parsec", update it with "mix deps.update nimble_parsec" or clean it with "mix deps.clean nimble_parsec"
    (stdlib 3.17.2.4) lists.erl:1358: :lists.mapfoldl/3
    (stdlib 3.17.2.4) lists.erl:1359: :lists.mapfoldl/3
    (stdlib 3.17.2.4) lists.erl:1358: :lists.mapfoldl/3
    (stdlib 3.17.2.4) lists.erl:1359: :lists.mapfoldl/3
    (elixir 1.11.4) expanding macro: Kernel.".."/2
    lib/nimble_parsec/compiler.ex:931: NimbleParsec.Compiler.newline_allowed?/1
  ```
so think it is better to set the minimum to elixir 1.12. wdyt?